sk-csharp-chatgpt-plugin/semantic-functions-generator/Models/AIPluginModel.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Newtonsoft.Json;
+
+namespace AIPlugins.AzureFunctions.Generator.Models;
+
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
+public class AIPluginModel
```

Comment:
Can this and `AIPluginSettings` be merged?

Reply:
AIPluginModel isn't even used; I deleted the file.